### ISSUE TITLE
Automated Test Suite: a11y 

### ISSUE THREAD

**Issue by rachelwhitton** [pantheon-systems/documentation#1243]
- [x] Warnings from a11y should fail build 
- [ ] Debug grunt results
 - [ ] Determine what accessibility criteria can/should be ignored for docs use case
